v7/_posts/release notes/2018-03-21-ODL-7.4.3.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "ODataLib 7.4.3"
+description: "ODataLib 7.4.3 release notes"
+category: "4. Release Notes"
+---
+
+## Changes in ODataLib 7.4.3 Release ##
+
+## Notes ##
+
+ODataLib 7.4.3 fixes a minor bug introduced in 7.4.2 in which the path for a contained entity set was computed incorrectly.
+
+## Fixed Bugs ##
+
+[[Issue #1121](https://github.com/OData/odata.net/issues/1121)] Incorrect path calculated for contained entity sets.
+
+---
+
+This release delivers OData core libraries including ODataLib, EdmLib, Spatial and Client.
```

Comment:
I am still have concern about a small bug in Core.lib will make us to release all of 4 libs.